### DISCUSSION TITLE
(Some of the) uploaded SVGs to markdown don't show correctly, when opening to new tab

### DISCUSSION THREAD

**Discussion by krisu5** [community/community#11053] [Other Feature Feedback, Questions, & Ideas]
**Example:**

![Github bg var](https://user-images.githubusercontent.com/3540275/152642951-ac1eaefa-6f57-44f8-a9a4-39c3081002e8.svg)

Works fine here, but open this to new tab and it's just black circle.

**Comment by TheMindVirus** [comment]
This is what it looks like the other way round, where the preview doesn't work but when you open it in new tab it appears.
https://github.com/themindvirus/liquorice/blob/main/liquorice.svg
![image](https://user-images.githubusercontent.com/8195854/164054534-dd911206-4a8c-47c2-9210-453bc657e376.png)
